src/components/Counter/docs/CounterExtendedExample.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import extendedStyles from './CounterExtendedExample.st.css';
+import { Counter } from '../Counter';
+
+interface State {
+  value: number;
+}
+
+export class CounterExtendedExample extends React.Component<{}, State> {
+  state = { value: 10 };
+
+  handleChange = (num: string) => {
+    let value;
+    try {
+      value = Number(num);
+      this.setState({ value });
+    } catch (e) {
+      console.log(num);
+    }
+  };
+
+  render = () => (
+    <Counter
```

Comment:
I think we just need to add a a disabled example here, so that we can see how changing the disabled color affects the component

Reply:
And maybe add a `min={-5}` and add a title saying:
`"Counter with a max value of 15, and a min value of -5"`

Reply:
done